### PR TITLE
ref(createProject.jsx)

### PR DESCRIPTION
**Pull request by jeffkwoh**
createProject.jsx is only used by views/newProject.jsx. Thus has been shifted into the same folder.

### CODE REVIEW

File: src/sentry/static/sentry/app/views/projectInstall/newProject.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import DocumentTitle from 'react-document-title';
 import React from 'react';
 import styled from '@emotion/styled';
 
-import CreateProject from 'app/components/createProject';
+import CreateProject from 'app/views/createProject';
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/projectInstall/newProject.jsx]: I don't think this is the correct path

Comment (jeffkwoh) [reply src/sentry/static/sentry/app/views/projectInstall/newProject.jsx]: That was embarrassing, fixed.

File: tests/js/spec/components/createProject.spec.jsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import {shallow, mountWithTheme} from 'sentry-test/enzyme';
 
-import {CreateProject} from 'app/components/createProject';
+import {CreateProject} from 'app/views/projectInstall/createProject';
```
Comment (billyvg) [inline tests/js/spec/components/createProject.spec.jsx]: Can we move this test file too so it matches the component?

Comment (jeffkwoh) [reply tests/js/spec/components/createProject.spec.jsx]: Done.

Comment (jeffkwoh) [reply tests/js/spec/components/createProject.spec.jsx]: done.

**Review by billyvg — APPROVED** [review]

**Pull request merged**